src/main/webapp/WEB-INF/views/addDriverToCar.jsp

```diff
@@ -0,0 +1,48 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Add driver</title>
+</head>
+<body>
+<h3>Add driver to car</h3>
+<form method="post" name="carDriverId">
```

Comment:
Optionally: perhaps you need to send this data to some page, but we should clarify with mentors  
```suggestion
<form method="post" name="${pageContext.request.contextPath}/pathToAddCar">
```